lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -181,6 +181,19 @@
         required: false
         default: null
         description: The CA bundle string with custom certificates. defaults to None.
+
+  auth_type:
+    required: false
+    description: Auth type endpoint configuration information. E.g. RSA key pair
```

Comment:
Thanks, better now, but still not perfectly clear.  Let's shoot for the goal that a user who never used ManageIQ UI could use this module based only on this documentation.
- What does "auth type endpoint" mean?  If I'm a user and have no idea there are Endpoint and Authentication objects behind this, what does this tell me?
- "E.g." — is that just one example, or is it always a key pair?
- I'm also confused between SSH and RSA.  Hmm, is it an SSH key pair of `rsa` type?

Reply:
Thanks for the review, really appreciated. I can see the confusion so have updated the commit which hopefully clarifies. Importantly I have switched from auth_type to ssh_keypair because, yes, its just the ssh keypair for connecting to Ironic nodes in OpenStack.